changelogs/fragments/5224-proxmox-unprivileged-default.yaml

```
@@ -0,0 +1,2 @@
+deprecated_features:
+  - proxmox - deprecated the current ``unprivileged`` default value, will be changed to ``true`` in 7.0.0 (https://github.com/pull/5224).
```

Since this also shows up in the Ansible combined changelog, it's better to say version 7.0.0 of what to make sure no confusion arises:
```suggestion
  - proxmox - deprecated the current ``unprivileged`` default value, will be changed to ``true`` in community.general 7.0.0 (https://github.com/pull/5224).
```